assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/AudienceTiles.js

```diff
@@ -508,6 +511,16 @@ export default function AudienceTiles( { Widget, widgetLoading } ) {
 						/>
 					);
 				} ) }
+				{ ! isTabbedBreakpoint && visibleAudiences.length === 1 && (
```

Comment:
Shouldn't we also display the `PlaceholderTile` on mobile and tablet screens?

Reply:
Sorry, we don't need to show it on the smaller screens, as there won't be a tab view for a single tile.